lib/datawow/models/moderations/moderation.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Datawow
+  # :nodoc:
+  class Moderation
+    include Datawow::Models::Interface
+
+    attr_writer :project_key
+
+    def initialize(token = nil)
+      @project_key = token
+      @type = :moderation
+      @query_str ||= nil
```

Comment:
what this line is doing?

Reply:
I actually dunno eiei

Reply:
I may? *refactor* this **later** since there are so many classes which have this line (I did copy this from others)

Reply:
I think we remove this
`@query_str ||= nil`

```
# frozen_string_literal: true

module Datawow
  module Models
    module Interface
      def all(options = {})
        connector.get(options)
      end

      def create(options)
        connector.post(options)
      end

      def find_by(options = {})
        @query_str = true if @query_str.nil?
        connector.get(options, @query_str)
      end

      private

      def connector
        @connection ||= Connector.new(@path, @type, token: @project_key)
      end
    end
  end
end
```

Reply:
should I remove every single `@query_str ||= nil` ?
or just this one

Reply:
@jsice I would left the others be still.